modules/openapi-generator/src/main/resources/python/python-experimental/model_utils.mustache

```diff
@@ -1043,6 +1050,13 @@ def get_anyof_instances(self, model_args, constant_args):
         return anyof_instances
 
     for anyof_class in self._composed_schemas()['anyOf']:
+        if anyof_class is none_type:
+            # The only way the input data can match the 'None' type
+            # is if the input data is the 'null' value.
+            if model_args is None:
```

Comment:
If model_args can be None please update our docstrings to show that
- here +
- get_oneof_instance
- any calling function that passes it in

Reply:
done